content/docs/command-reference/plots/modify.md

```diff
@@ -0,0 +1,133 @@
+# plots modify
+
+Modify display properties of [plot metrics](/doc/command-reference/plots) files.
+
+## Synopsis
+
+```usage
+usage: dvc plots modify [-h] [-q | -v] [-t <name_or_path>] [-x <field>]
+                        [-y <field>] [--no-csv-header] [--title <text>]
+                        [--x-label <text>] [--y-label <text>]
+                        [--unset [<prop> [<prop> ...]]]
+                        target
+
+positional arguments:
+  target                Metric file to set props to
+```
+
+## Description
+
+It might be not convenient for users or automation systems to specify all the
+_display properties_ (such as `y-label`, `template`, `title`, etc.) each time
+plots are generated with `dvc plot show` or `dvc plot diff`. This command sets
+(or unsets) default display props for specific plot files.
+
+The required `target` metrics file should be an <abbd>output</abbr> of one of
```

Comment:
I would try to simplify this paragraph (and similar ones in other docs - too much references to ... output ... pipeline ... listed - it's scary. We can just explain simpler:

`target` is a required path to a metric file (must be set in `dvc.yaml`, see `--plot` for the `dvc run`) to change settings for. 

WDYT?

Reply:
Yes, looks good. I had to change it a little but updated this in b0f49c36